.clang-tidy

```diff
@@ -19,7 +19,7 @@ Checks:
 '
 WarningsAsErrors:  '*'
 HeaderFilterRegex: '.*'
-ExcludeHeaderFilterRegex: 'include/lmdb.h|Grammar.h|dbus/*'
+ExcludeHeaderFilterRegex: 'include/lmdb.h|include/python*/*|Grammar.h|dbus/*'
```

Comment:
The addition is necessary on macos+brew. Without it, clang-tidy checks the Python header files.